src/app/dropdown/dropdown-angular-nested.demo.html

```diff
@@ -16,16 +16,16 @@
             <a href="javascript://" clrDropdownItem>Action 1</a>
             <a href="javascript://" class="disabled" clrDropdownItem>Disabled Action</a>
             <div class="dropdown-divider"></div>
-            <clr-dropdown [clrMenuPosition]="'right-bottom'">
+            <clr-dropdown>
                 <button clrDropdownToggle>Link 1</button>
```

Comment:
So we talked about this on Slack. We have an issue with different types of triggers, but I just realized we could have `clrClickTrigger` and `clrHoverTrigger`. Not sure if it's worth it right now. And once again, I think we can do it without breaking changes.